src/nms.cc

```diff
@@ -0,0 +1,86 @@
+#include <cmath>
+#include <algorithm>
+
+namespace nms
+{
```

Comment:
code style in Paddle repo is Google C++ code style:

```
namespace nms {

}
```

“The open curly brace is always on the end of the last line of the function declaration, not the start of the next line.”

https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions

Reply:
done